Shootingat Resolved Project/Assets/Scripts/Characters/Player/Powerups/HealthPowerup.cs

```diff
@@ -8,12 +8,12 @@ public class HealthPowerup : Powerup
     {
         [SerializeField] private int addedHealth;
 
-        public override void ApplyPowerup()
+        public override void ApplyPowerup(PlayerStats ps)
         {
             if (goodPowerup)
-                PlayerStats.Instance.ModifyClarity(addedHealth);
```

Comment:
Split method to be more clear about whether it is improving it or making it worse